.gitignore

```diff
@@ -3,3 +3,4 @@
 benches.json
 target
 deno.lock
+.vscode/settings.json
```

Comment:
let's just make this `.vscode` to ignore the entire dir. You can then delete the dir

Reply:
ok, changed